x-pack/plugins/ml/public/application/model_management/deployment_setup.tsx

```diff
@@ -369,21 +631,33 @@ export const StartUpdateDeploymentModal: FC<StartDeploymentModalProps> = ({
   startModelDeploymentDocUrl,
   initialParams,
   modelAndDeploymentIds,
+  cloudInfo,
+  deploymentParamsMapper,
+  showNodeInfo,
 }) => {
-  const { showNodeInfo } = useEnabledFeatures();
-
   const isUpdate = !!initialParams;
 
-  const { total_ml_processors: totalMlProcessors } = getNewJobLimits();
+  const getDefaultParams = (): DeploymentParamsUI => {
```

Comment:
Nit, could be wrapped in a `useCallback`

Reply:
updated in a7b1feaa90a7658af57929ac653741bccc3bfb3a